charts/grafana/templates/pvc.yaml

```
@@ -6,6 +6,9 @@ metadata:
   namespace: {{ template "grafana.namespace" . }}
   labels:
     {{- include "grafana.labels" . | nindent 4 }}
+    {{- if .Values.persistence.extraPvcLabels }}
+    {{- toYaml .Values.persistence.extraPvcLabels | nindent 4 }}
+    {{- end }}
```

Since by default `.Values.persistence.extraPvcLabels` is `{}`, we should avoid it to show up.

```suggestion
    {{- with .Values.persistence.extraPvcLabels }}
    {{- toYaml . | nindent 4 }}
    {{- end }}
```